Makefile

```diff
@@ -1,3 +1,17 @@
+# Copyright 2021 The Kubernetes Authors.
```

Comment:
Ianal, is this what we want here? `The KCP Authors`?

Reply:
/shrug

Open to ideas, no clue